packages/react/src/components/ValueCard/Attribute.jsx

```diff
@@ -162,20 +162,22 @@ const Attribute = ({
             data-testid={`${testId}-secondary-value`}
             className={`${BEM_BASE}-secondary-value`}
             style={{
-              '--secondary-value-color': secondaryValue.color || '#777',
+              '--secondary-value-color': '#6F6F6F',
```

Comment:
We should bring in the carbon color tokens `gray60`. You can import from @carbon/colors package
`import { gray60 } from '@carbon/colors'`

Reply:
@davidicus ,Thanks for reviewing it.
Code has been updated.

https://deploy-preview-2878--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-valuecard--with-trends